src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
 body {
   margin: 0;
+  width: 1024px;
+  height: 1044px;
```

Comment:
Similarly, setting a fixed height on the body element can lead to layout issues on different devices. Consider using a responsive design approach instead.